cypress/e2e/manager_login.cy.js

```diff
@@ -0,0 +1,28 @@
+import testUser from "../../credentials/managerCredentials.json"
+import LoginPage from "../pages/LoginPage.js"
+
+describe('tests for login function', () => {
+  beforeEach(() => {
+    cy.visit('/Account/Login?ReturnUrl=%2F')
+  })
+  it('Negative scenario - try to login without credentials', () => {
+    LoginPage.loginButton().click()
+    LoginPage.emptyLoginFieldMessageCheck()
+  })
+
+  it('Negative scenario - login with incorrect credentials', () => {
+    LoginPage.submitLogin(testUser.incorrectLogin,testUser.incorrectPass)
+    LoginPage.incorrectLoginFieldMessageCheck()
+  })
+
+
+  it('Positive scenario - login with correct credentials', () => {
+    LoginPage.submitLogin(testUser.login,testUser.pass)
+  
+  }
+  )
+
+  // it('Positive scenario - login with correct credentials', () => {
```

Comment:
jezeli wrzucasz jakis kod do repo ktory jest zakomentowany to w pierwszej linii napisz dlaczebo np bo bedzie potrzebny w przyszlosci czy feakture aktualnie nie dziala ale ktos go poprawia - jezeli to jest do wywalania to usun z prki

Reply:
ok